src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationHubLinks.java

```diff
@@ -1,6 +1,10 @@
 package com.wikia.webdriver.testcases.globalnavigationtests;
 
+import org.openqa.selenium.WebDriver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.WebDriver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)